Otherworldly Pines/Assets/Scripts/DialogTrigger.cs

```diff
@@ -27,9 +27,13 @@ public class DialogTrigger : MonoBehaviour
 
             var gravityControls = player.GetComponent<GravityControl>();
             var throwControls = player.GetComponent<PlayerThrow>();
-            playerControls.FreezeControls();
-            gravityControls.FreezeControls();
-            throwControls.FreezeControls();
+            if (!GameObject.FindGameObjectWithTag("CM").GetComponent<CheckpointMaster>().skipTutorial)
+            {
+                playerControls.FreezeControls();
+                gravityControls.FreezeControls();
+                throwControls.FreezeControls();
+            }
+           
```

Comment:
Instead of this, could we just return at the beginning of the function if we are skipping tutorials? This still misses a few other things that happen in the function

Reply:
Nvm, I added something to this PR so I just did this real quick too